.gitlab-ci.yml

```
@@ -105,6 +105,25 @@ publish-draft-release:
   interruptible:                   true
   allow_failure:                   true
 
+test-deterministic-wasm:           &test-linux
```

```suggestion
test-deterministic-wasm:
```

No need to add an anchor here, since we're not using this job in other jobs.